go.sum

```diff
@@ -80,6 +80,8 @@ github.com/digitalocean/ripeatlas v0.0.0-20210504234211-903c81b0862a h1:J1i9WX5T
 github.com/digitalocean/ripeatlas v0.0.0-20210504234211-903c81b0862a/go.mod h1:SvVsPwhes3CGTzLE3btxIfYyF/jJhoMYN5Zb/aWjBks=
 github.com/digitalocean/ripeatlas v0.0.0-20210505051840-4ed087e8eb1c h1:vHq59WfqS+sXJhM4AEB86TBVIakIiW3Xk2ambwlQPyo=
 github.com/digitalocean/ripeatlas v0.0.0-20210505051840-4ed087e8eb1c/go.mod h1:SvVsPwhes3CGTzLE3btxIfYyF/jJhoMYN5Zb/aWjBks=
```

Comment:
I wonder if the replace directive will catch these other indirect references?

Reply:
It looks like replace supports version agnostic replacements https://golang.org/doc/modules/gomod-ref#replace-examples

Reply:
That's interesting. It looks to me like it's just preserving the hash of the previous versions, for whatever reason. I don't see why we would ever reference those old commits.

I'm not entirely clear on how we would update this to be version agnostic though. The examples on that page seem to be pointing to specific version tags in the forked repository. (I was avoiding doing any tagging because I wasn't sure how that would integrate with the unversioned upstream.)

What would you suggest? We _could_ start tagging the `ripeatlas` repository, if we want to get away from these difficult-to-use strings. Right now my process is just "replace the `v0.0.0-<date>-<version>` string with `master` and re-run `go mod download`".

Reply:
I went ahead and ran `go mod tidy` and it removed all the obsolete references. I guess I'll add that to my process. 🙂 TIL; thanks!